### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by v-demjoh**
## Purpose
Found and improved broken links.

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: quickstart/csharp/uwp/translate-speech-to-text/README.md [hunk 1]
```diff
@@ -35,5 +35,5 @@ When you see the `Enable microphone` button, click it. Then select the `Translat
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-uwp)
+* [Get started with speech translation](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-csharp)
```
Comment (yulin-li) [inline quickstart/csharp/uwp/translate-speech-to-text/README.md:38]: need to remove `en-us`

File: quickstart/cpp/windows/translate-speech-to-text/README.md [hunk 2]
```diff
@@ -36,5 +36,5 @@ To debug the app and then run it, press F5 or use **Debug** \> **Start Debugging
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-cpp-windows)
+* [Get started with speech translation](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-cpp)
```
Comment (v-demjoh) [inline quickstart/cpp/windows/translate-speech-to-text/README.md:39]:
```suggestion
* [Get started with speech translation](/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-cpp)
```

File: quickstart/csharp/dotnet/translate-speech-to-text/README.md [hunk 3]
```diff
@@ -32,5 +32,5 @@ To debug the app and then run it, press F5 or use **Debug** \> **Start Debugging
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-dotnetframework-windows)
+* [Get started with speech translation](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-csharp)
```
Comment (v-demjoh) [inline quickstart/csharp/dotnet/translate-speech-to-text/README.md:35]:
```suggestion
* [Get started with speech translation](/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-csharp)
```

File: quickstart/csharp/uwp/translate-speech-to-text/README.md (hunk 1 above)
Comment (v-demjoh) [inline quickstart/csharp/uwp/translate-speech-to-text/README.md:38]:
```suggestion
* [Get started with speech translation](azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-csharp)
```

File: quickstart/java/jre/translate-speech-to-text/README.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ See the [accompanying article](https://docs.microsoft.com/azure/cognitive-servic
 
 ## Build the sample
 
-> Note: more detailed step-by-step instructions are available [here](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-java-jre).
+> Note: more detailed step-by-step instructions are available [here](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-java).
```
Comment (v-demjoh) [inline quickstart/java/jre/translate-speech-to-text/README.md:37]:
```suggestion
> Note: more [detailed step-by-step instructions are available here](/azure/cognitive-services/speech-service/get-started-speech-translation?tabs=script%2Cwindowsinstall&pivots=programming-language-java).
```

File: samples/csharp/unity/VirtualAssistantPreview/README.md [hunk 5]
```diff
@@ -122,5 +122,5 @@ Start by running it in the Unity Editor (first option), then explore other optio
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-unity)
+* [Quickstart: Setup development environment](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstarts/setup-platform?tabs=unity%2Cwindows%2Cjre%2Cbrowser&pivots=programming-language-csharp)
```
Comment (v-demjoh) [inline samples/csharp/unity/VirtualAssistantPreview/README.md:125]:
```suggestion
* [Quickstart: Setup development environment](/azure/cognitive-services/speech-service/quickstarts/setup-platform?tabs=unity%2Cwindows%2Cjre%2Cbrowser&pivots=programming-language-csharp)
```

**Review by BrianMouncer — APPROVED** [review]

**Comment by v-demjoh** [comment]
#do-not-merge

Please instead merge https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/1240

**Pull request closed without merging**